src/MUIRichTextEditor.tsx

```diff
@@ -1010,6 +1028,7 @@ const MUIRichTextEditor: RefForwardingComponent<TMUIRichTextEditorRef, IMUIRichT
                     setSelectedIndex(limit - 1)
                 }
                 return "mui-autocomplete-navigate"
+            //TODO: Fix 'Enter' key event. Function isn't currently recieving 'enter' key
```

Comment:
there was a merge from gitstart that will have changed this behaviour

Reply:
Resolved/Fixed